### DISCUSSION TITLE
Best approach for languages not handled by Weblate (but by maintainers themselves)

### DISCUSSION THREAD

**Discussion by papjul** [WeblateOrg/weblate#9703] [Q&A] [answered]
Hi,

I’m a maintainer of an Android app (using Hosted Weblate for FOSS projects), which has English as its base language, and also has a French translation, that is my native language. So I know the context of any string better than anyone else and I also have years of experience in translation as a hobby, so I know how to do correct wording in French. For these reasons, I don’t want to open French language to Weblate and I’m not open to suggestions (if someone really thinks something need to be changed, they can still reach me on the project issues page).

So to do that I used the "Language regex":
https://docs.weblate.org/en/latest/admin/projects.html#component-language-regex

As suggested in the doc, it can be used "to limit the list of languages managed by Weblate" and I found no other alternative (but as there are a lot of options, I may have missed it).

However, this approach makes the French language appear as missing (0 % completed), and Weblate even suggests me to add French in the project when I log in (“This translation does not exist yet”).

This issue doesn’t appear with the base language (English). I’m able to check “Edit base file” option to open English to suggestions (as a non-native English speaker, I’m open to it), but if I don’t open it, English language still appears as completed even though it’s not open to translators.

Is there any way to do something similar to what you can do with base language? (that would be the easiest and cleanest for me) Or else, what would you suggest? I’m looking for a “set up once and forget” solution.

Thank you for this wonderful product, it helped our project a lot getting more translations included and updated!

**Comment by nijel** [comment]
You probably have the language present in some other component (most likely glossary)? Removing it there should address it.

**Reply by papjul** [reply]
I deleted French translation on Glossary and now it is suggesting me to contribute to French translation on Android project AND on glossary, so I'm not exactly sure to understand what it fixed.

Link to the project if that helps: https://hosted.weblate.org/projects/breezy-weather/

**Reply by nijel** [reply, marked as answer]
Ah, language regex is not applied to suggested translations, I've created https://github.com/WeblateOrg/weblate/issues/9714 to track this.
